osu.Game/Screens/SelectV2/SongSelect.cs

```diff
@@ -146,9 +146,9 @@ private void load()
                                     RelativeSizeAxes = Axes.Both,
                                     ColumnDimensions = new[]
                                     {
-                                        new Dimension(GridSizeMode.Relative, 0.5f, maxSize: 850),
+                                        new Dimension(GridSizeMode.Relative, 0.5f, maxSize: 660),
                                         new Dimension(),
-                                        new Dimension(GridSizeMode.Relative, 0.5f, maxSize: 750),
+                                        new Dimension(GridSizeMode.Relative, 0.5f, maxSize: 580),
```

Comment:
Definitely one of the changes that looked very unsettling to me on first glance but maybe that's aggravated by me using an ultra wide display:

![osu_2025-05-29_13-01-05](https://github.com/user-attachments/assets/91efeb83-2e91-4985-8cc2-41f3569984e4)

but after a few minutes of staring at it maybe I got used to it. Willing to see how it shakes out.

Reply:
I think we may want a second level of rules for ultrawidescreen... i get the concern but i think it's an edge case and still looks okayish.